communitiesuk/funding-service-design-workflows
# FS-3535: Creation of additional SQS Queue and S3

##Change description
Notification service will have a subcriber service which continuously read data from the SQS service. An extended client will be implemented to use both the S3 and SQS services.

☑︎ Unit tests and other appropriate tests added or updated
☑︎ README and other documentation has been updated / added (if needed)
☑︎ Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

How to test
Log in into the aws console and check the SQS Queue console and S3 console to verify that the resources have been created.
